js/components/main.js

```diff
@@ -511,6 +511,10 @@ class Main extends ImmutableComponent {
 
     const onWindowMove = debounce(function (event) {
       const position = event.sender.getPosition()
+      // Windows may be placed off-screen
+      // See https://github.com/brave/browser-laptop/issues/3558
+      position[0] = Math.max(0, position[0])
```

Comment:
should we be preventing partial off-screen placement? Seems like this could cause weird issues with multi-monitor placement or intentionally placing windows a little out of the way. I believe you can use the screen api to determine if a particular point is on any screen and I think we should probably use that to make sure that some portion of the draggable area is on the screen